app/views/external_users/claims/case_details/_fields.html.haml

```diff
@@ -2,24 +2,27 @@
   - if @claim.agfs? && @external_user.admin?
     = render partial: 'external_users/claims/case_details/advocate_fields', locals: { f: f }
 
-  = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter, hint_text: t('common.external_user.providers_ref_hint')
+  = f.govuk_text_field :providers_ref,
+    label: { text: t('common.external_user.providers_ref') },
+    hint: { text: t('common.external_user.providers_ref_hint') },
+    width: 'one-half'
 
   - if @claim.requires_case_type?
     = render partial: 'external_users/claims/case_details/case_type_fields', locals: { f: f }
 
-  #cc-court.form-group{ class: error_class?(@error_presenter, :court) ? 'form-group-error dropdown_field_with_errors' : '' }
-    %label.form-label-bold{ for: 'court' }
-      = t('.court')
-      .form-hint.xsmall.zero-vert-margin
-        = t('.court_hint')
-      = validation_error_message(@error_presenter, :court)
+  #cc-court.fx-autocomplete-wrapper
+    = f.govuk_collection_select :court_id,
+      Court.alphabetical,
+      :id,
+      :name,
+      options: { include_blank: '' },
+      label: { text: t('.court') },
+      hint: { text: t('.court_hint') }
 
```

Comment:
same as above. this can be written as 
`form_group: { classes: [ 'fx-autocomplete-wrapper' ], id: 'cc-court' }`